frontends/web/src/routes/buy/exchange.tsx

```diff
@@ -50,9 +50,9 @@ type TOption = {
     value?: string;
 }
 
-const SelectedRegionIcon = ({ regionCode }: { regionCode: string }) => {
+const SelectedRegionIcon = ({ regionCode, noRegion }: { regionCode: string, noRegion: string }) => {
   return <span className={style.flag}>
-    {regionCode === '' ? <Globe /> : <span className={`fi fi-${regionCode}`}></span>}
+    {regionCode === noRegion ? <Globe /> : <span className={`fi fi-${regionCode}`}></span>}
   </span>;
```

Comment:
oups, I wanted to play around if we could still keep empty string to select the globe and now ended up with an alternative approach. 

Instead of changing noRegion to dash wouldn't something like this work as well instead of this PR?

```diff
diff --git a/frontends/web/src/routes/buy/exchange.tsx b/frontends/web/src/routes/buy/exchange.tsx
index 34b85e729..80ff74167 100644
--- a/frontends/web/src/routes/buy/exchange.tsx
+++ b/frontends/web/src/routes/buy/exchange.tsx
@@ -93,6 +93,10 @@ export const Exchange = ({ code, accounts }: TProps) => {
       setSelectedRegion(config.frontend.selectedExchangeRegion);
       return;
     }
+    // user had selected no region before, do not continue to guessing region
+    if (config.frontend.selectedExchangeRegion === '') {
+      return;
+    }
 
     const userRegion = getRegionNameFromLocale(nativeLocale || '');
     //Region is available in the list
@@ -198,7 +202,6 @@ export const Exchange = ({ code, accounts }: TProps) => {
                       className={style.extraLeftSpace}
                       options={[{
                         text: t('buy.exchange.selectRegion'),
-                        disabled: true,
                         value: '',
                       },
                       ...regions]
@@ -221,7 +224,6 @@ export const Exchange = ({ code, accounts }: TProps) => {
                   <div>
                     {!noExchangeAvailable && allExchangeDeals && allExchangeDeals.exchanges.map(exchange => exchange.supported && (<ExchangeSelectionRadio
                       key={exchange.exchangeName}
-                      disabled={!selectedRegion}
                       id={exchange.exchangeName}
                       exchangeName={exchange.exchangeName}
                       deals={exchange.deals}
```

Reply:
Very nice! LGTM :) I pushed an updated following your approach, thanks!